### PR TITLE
Update README.md to update guiding principles

### PR DESCRIPTION
**Pull request by malvikasharan**
We had a team day this week where we discussed some principles that informs the updates in our guiding principles - suggested in this PR.

### CODE REVIEW

**Review by cassgvp — APPROVED** [review]
Looks great.

**Review by EKaroune — APPROVED** [review]
Thanks for updating @malvikasharan 

Looks good to me

**Comment by malvikasharan** [comment]
@GabinWK @eirini-zormpa @aleesteele @vhellon - I will wait for each of your comments before merging this to be sure that you have read this and any of your concerns are addressed before this becomes integrated in our guiding principles.

**Comment by GabinWK** [comment]
I was wondering whether we could have those principles numbered and categorised. That would look more structured then itemised and perhaps easier to present and maybe remember (?) 
Something in the line of " Our 10 guiding principles are ..."   10 just like 3 (maybe 5 too) is a nice number to remember :-)

10 PRINCIPLES

Social 
1.	Maintain respect, empathy and integrity in the team and the communities we build.
2.	Prioritise diversity, equity and inclusion in all our work.
3.	Create a safe and non-judgemental environment to engage each other, celebrate successes, admit errors and draw learnings from our mistakes.
4.	Be willing to work through conflicts to resolution.

Work?
5.	Share knowledge about the scientific communities you coordinate and the issue you are seeking to address.
6.	Provide access to resources that empower and enable each other to discover, display and directly use their knowledge and skills.
7.	Ensure that transparent reporting practices and effective communications become a regular and ongoing component of our community-building process.
8.	Collaborate with individuals and institutions beyond the team that expand and extend our mission for open Research Community Management.

Growth?
9.	Actively create and identify future opportunities to enable collaboration and integration of open research practices to co-create additional knowledge.
10.	Contribute to the ongoing professional development of team members by maximising opportunities for collaborative work, their careers, and open research at large.

**Comment by malvikasharan** [comment]
Thanks Gabin. I think for the Turing values we should not used numbered list as it gives notion of hierarchy or priority in which we follow them. Also, that will keep it consistent with the Turing’s format.

For the other principles, I don't see them as applied to specific contexts. But I am happy to hear what other think and go with the majority.

File: README.md [hunk 1]
```diff
@@ -116,28 +124,31 @@ You can also contact [Malvika Sharan](mailto:msharan@turing.ac.uk) for any quest
...
+- **Inclusivity**: Embed equity, diversity, inclusion and accessibility (EDIA) approaches in all our work.
+- **Respect**: Maintain respectful co-creation processes in the team and the communities by providing clear ways to participate, engage and lead.
+- **Trust**: Build trust in the community by making implicit knowledge explicit and providing access to resources that enable everyone to discover, display and directly use their knowledge and skills. 
+- **Integrity**: Ensure ethical integrity through involvement of community members, especially those impacted by our work.
+
+## Guiding Principles for the team
+
+1. **Collaboration**: Actively break down silos by enabling knowledge exchange across all projects through collaboration and joint efforts.
+1. **Communication**: Communicate openly the Communities of Practice you manage, challenges you are seeking to address and solutions that have emerged.
+1. **professionalisation**: Uphold the highest personal and professional standards and share opportunities to co-create knowledge and infrastructure around community management and open research practices as a team.
```
Comment (vhellon) [inline README.md:140]:
```suggestion
1. **Professionalisation**: Uphold the highest personal and professional standards and share opportunities to co-create knowledge and infrastructure around community management and open research practices as a team.
```

**Comment by vhellon** [comment]
Looks good to me too

File: README.md (hunk 1 above)
Comment (malvikasharan) [reply README.md]: Thanks for spotting that Vicky.

**Comment by malvikasharan** [comment]
@GabinWK I have added bullet points now - that helped remove some redundancies too. Thank you for the feedback.

File: README.md [hunk 2]
```diff
@@ -156,7 +171,7 @@ Their engagement with *The Turing Way* project and community should help them un
 
 ## References
 
-- _Sharan, Malvika, & Mehonic, Aida. (2022, July 21). Community Managers and Research Application Managers at the Tools, Practices and Systems Programme at The Alan Turing Institute. Zenodo. https://doi.org/10.5281/zenodo.6878124_
```
Comment (aleesteele) [inline README.md:159]: Is this reference not going to be added?

Comment (malvikasharan) [reply README.md:159]: This is an outdated reference - I think we should upload this in the CM resources (it is already in our shared drive) but not any longer suitable as a main source.

**Comment by malvikasharan** [comment]
Assuming that all of you have read this. I will merge and share with the TPS team. Thank you all.

**Pull request merged**